typings/index.d.ts

```diff
@@ -281,6 +281,9 @@ export class Command {
   processedArgs: any[];
   readonly commands: readonly Command[];
   readonly options: readonly Option[];
+  readonly registeredArguments: readonly Argument[];
+  /** @deprecated Use `.registeredArguments` instead. */
```

Comment:
Use same style as other deprecated messages
```suggestion
  /** @deprecated since v11, instead use `.registeredArguments` */
```

Reply:
I actually thought about specifying the version, but decided against it because the property had never been supported officially.